src/gen.rs

```diff
@@ -112,12 +112,13 @@ impl Generator {
                 }
 
                 Schema::Union(ref union) => {
+
```

Comment:
Same small blank and ; comments for this function

Reply:
:+1: